amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/README.md

```diff
@@ -25,17 +35,144 @@ Input files are in the [input_files](input_files) directory.  There are two stag
 
 Current case was run with AMR-Wind version [f67a52dd6aa1882595d16700527470bc8097cb13](https://github.com/Exawind/amr-wind/commit/f67a52dd6aa1882595d16700527470bc8097cb13)
 
+## Performance
+
+Full details provided in [**performance documentation**](performance/README.md).
+
+The simulation was run on the Sandia Flight HPC cluster using the following resources: 
+
+| Parameter       | Value |
+|---              |---  |
+| Number of nodes | 8   |
+| Number of CPUs  | 896 |
+| Wall-time       | 5.4 hours|
+| CPU-hours       | 4821.7 |  
+
 
 ## Results
 
-Statistics and results from the ABL are calculated using scripts and notebooks from the [postprocessing](postprocessing) directory, and saved to the [results](directory).  These include quantities such as the:
+Statistics from the ABL are calculated over the time interval $t \in [15000 , 20000]s$ using scripts and notebooks from the [postprocessing](postprocessing) directory. The results are saved to the [results](results/) directory.  
+
+Horizontally averaged statistics are computed in the
+[AVG_horiz_profiles.ipynb](postprocessing/AVG_horiz_profiles.ipynb) notebook and
+are summarized in the following table at the bottom-tip, hub-height, and top-tip
+location of the NREL5MW:
+
+| z | Horizontal Velocity | Wind Direction | TI (TKE) | Shear Exponent | Veer|
+|--|--|--|--|--|--|
+|27 m  | 9.89 m/s  | 239.74 deg | 8.29 % | 0.1300 | 0.0087 deg/m
+|90 m  | 11.40 m/s  | 240.00 deg | 6.17 % | 0.0815 | 0.0006 deg/m
+|153 m  | 11.90 m/s  | 240.02 deg | 5.38 % | 0.0693 | 0.0001 deg/m
+
+The veer and shear exponent over the rotor disk are 0.0015 deg/m and 0.0944, respectively. The corresponding friction velocity, $u^*$, inversion height, $z_i$, and Obukhov length-scale for this simulation are:
+
+ - $u^*$ = 0.468 m/s
+ - $z_i$ = 803.133 m
+ - Obukhov Length = -2447.446 m (computed at z=27 m)  
```

Comment:
Noticed this was negative.
Maybe that's right, I have no idea haha.

Reply:
Yeah, that's consistent.  Obukhov length is negative for unstable ABL's, and positive for stable ABL's.  The sign comes from the temperature gradient in the calculation.